packages/lit-dev-content/site/playground/signin.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="robots" content="noindex" />
+  </head>
+  <body>
+      <!-- In the future, this page will communicate with its parent Playground
+           page (having been opened as a popup window) to pass back an
```

Comment:
Non blocking, just for my understanding: as an iframe?

Reply:
It will be a popup window. Next PR will demonstrate!